cloud.foundry.cli/src/main/java/cloud/foundry/cli/crosscutting/logging/Log.java

```diff
@@ -66,20 +84,65 @@
         }
     }
 
+    private Log(String name) {
+        logger = Logger.getLogger(name);
+
+        // we want to enable really fine logs on this logger by default
+        // log level filtering is done in the baseLogger
+        logger.setLevel(DEBUG_LEVEL);
+    }
+
+    /**
+     * Factory method. Returns log object for a given name. New log objects will be created on demand.
+     */
+    public static Log getLog(String name) {
```

Comment:
This method could be set to private as it is not used anywhere else. Also the documentation should explain in which cases an exception is thrown.

Reply:
It's used by the tests for example, so clearly used somewhere. Not sure if I could hack the tests to use the other factory.

I'll update the docs.